.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: macOS-11
```

Comment:
Putting this here so I have a thread. In my PR review I asked:

> I also wanted to know: do we have confirmation that this PR resolved https://github.com/dfed/CacheAdvance/pull/67#issuecomment-1275266242?

I am having trouble following what exactly we think fixed the codecov issue ultimately and how we're confident that it's resolved. I'm following up because I want to understand better. Thanks for all of your work here @dfed !

Reply:
Oh, what fixed the codecov issue was:
1. Updating to a newer (non-deprecated) version of codecov
2. Manually creating the `lcov` file that codecov could pick up (since the newer version of codecov couldn't find the coverage files in derived data either)

Based on the logs from recent builds – the deprecated version of codecov wasn't picking up the `xcresult` file that includes coverage information. I probably could have fixed the issue by manually creating the `lcov` file like I did in this MR, but I figured I should start by updating to a non-deprecated tool.

You can see from [recent logs](https://github.com/dfed/CacheAdvance/actions/runs/3237806944/jobs/5305302300#step:8:30) that we are finding+uploading the `lcov` files. Looking through other recent builds ([all builds are here](https://github.com/dfed/CacheAdvance/actions)), you can see the same logs.

The very high-level is: codecov was failing to find our coverage data with the old config, and moving to this new config (namely, making the `lcov` files for codecov rather than making codecov create these files from our derived data) fixed that.

I still do not understand why neither the deprecated tool nor the modern tool could create these coverage files given the inputs we were giving them. But at least we have a workaround.

Reply:
Thank you for explaining, and for the log pointer!